lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -1078,8 +1078,8 @@ def compute_timestamp(record, time)
       elsif @service_name == CLOUDFUNCTIONS_SERVICE &&
             @cloudfunctions_log_match
         timestamp = DateTime.parse(@cloudfunctions_log_match['timestamp'])
-        ts_secs = timestamp.strftime('%s')
-        ts_nanos = timestamp.strftime('%N')
+        ts_secs = Integer(timestamp.strftime('%s'))
```

Comment:
I looked it up. Seems `to_i` will convert strings like 'cat' to 0, while Integer(some_string) will error out. So I've changed this to use `to_i` instead.